### PR TITLE
Improves description querying and formatting

### PR DESCRIPTION
**Pull request by gordania**
Closes issue #3

### CODE REVIEW

File: previews/model.py [hunk 1]
```diff
@@ -56,10 +56,17 @@ def fetch(self):
         self.title = titles[0].strip()
 
         # Get the desc from whatever we can find
-        desc_elems = soup.findAll(attrs={"name": re.compile(r"Desc", re.I)})
+        desc_elems = [soup.findAll(attrs={attr: re.compile(r"Desc", re.I)}) for attr in ["name", "property"]]
 
-        if len(desc_elems) > 0:
-            self.desc = desc_elems[0]["content"]
+        for i in [0, 1]:
```
Comment (davidhariri) [inline previews/model.py]: try out `for i in range(1):`. It generates an iterable and looks cooler 🛴

File: previews/model.py [hunk 2]
```diff
@@ -56,10 +56,17 @@ def fetch(self):
...
+            if len(desc_elems[i]) > 0:
+                self.desc = desc_elems[i][0]["content"]
+                break
+            else:
+                self.desc = ""
+
+        if len(self.desc.split()) > 30:
+            self.desc = " ".join(self.desc.split()[0:29]).strip()
```
Comment (davidhariri) [inline previews/model.py]: What about checking for `...` at the end of the string and, if not present, adding it in? I'm 99% sure RTM did that. Take a look at that codebase. I'd copy it in to this file, comment it out, and go line by line making sure all the features from it are in here and work as expected 👍 ☺️

File: previews/model.py (hunk 1 above)
Comment (gordania) [inline previews/model.py]: 👍

File: previews/model.py [hunk 3]
```diff
@@ -56,10 +56,20 @@ def fetch(self):
...
+            
+            if re.search(r"(\.\.\.|…)", self.desc) is None:
+                self.desc = self.desc + "..."
```
Comment (davidhariri) [inline previews/model.py]:
```py
something = " ….12...345..…  "

something = something.strip()
something = something.strip("…")
something = something.strip(".")

something += "..."

print(something)
```

**Pull request merged**